arangod/Cluster/ClusterInfo.cpp

```diff
@@ -6454,8 +6454,9 @@ ClusterInfo::SyncerThread::~SyncerThread() { shutdown(); }
 bool ClusterInfo::SyncerThread::notify() {
   std::lock_guard<std::mutex> lck(_m);
   _news = true;
+  // TODO: can we move the notify_one() call outside of the mutex?
```

Comment:
I looked code of class and yes we can. Ofc there wasn't big difference because pthread condvar/mutex aware about notification under lock